CMakeLists.txt

```diff
@@ -180,11 +180,11 @@ if(${DOXYGEN_FOUND})
   # Optionally download Doxygen tagfiles for dependencies.
   set(DOXYGEN_TAGFILES "")
   if(DOWNLOAD_TAGFILES)
-    set(tagfile_url "http://dartsim.github.io/dart/v6.3.0/dart.tag")
+    set(tagfile_url "http://dartsim.github.io/dart/v6.6.2/dart.tag")
```

Comment:
I wonder if there's a way to auto-generate this version from the `find_package` invocation above, so there's one fewer checkbox to remember when updating DART versions.

Reply:
Yep, we an use ${DART_VERSION}